### PR TITLE
Fix listingpost localebug

### PR DESCRIPTION
**Pull request by ovan**

### CODE REVIEW

File: app/views/listings/form/_price.haml [hunk 1]
```diff
@@ -33,7 +33,7 @@
                 "#fee_display",
                 "#listing_currency",
                 #{commission_from_seller},
-                #{minimum_commission},
+                #{minimum_commission.format(decimal_mark: ".", symbol: false)},
```
Comment (rap1ds) [inline app/views/listings/form/_price.haml]: This is not right for all languages. Finnish for example uses `,` as a decimal mark.

**Comment by sktoiva** [comment]
:+1:

File: app/views/listings/form/_price.haml (hunk 1 above)
Comment (ovan) [inline app/views/listings/form/_price.haml]: Yep, that's the point. We are formatting number for JS to consume and localization messes things up. :)

Comment (rap1ds) [inline app/views/listings/form/_price.haml]: ...and I just realized that this value is probably passed to JS as an integer and thus my comment should be ignored.

**Pull request merged**